libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -434,41 +438,69 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
-  private readonly elementToParentMap: { [key: string]: () => HTMLElement } = {
-    'KIRBY-MODAL-FOOTER': () => this.elementRef.nativeElement,
-    'KIRBY-PAGE-TITLE': () => this.ionTitleElement.nativeElement,
-    'KIRBY-PAGE-PROGRESS': () => this.ionToolbarElement.nativeElement,
+  private readonly elementToParentMap: { [key: string]: () => HTMLElement[] } = {
+    'KIRBY-MODAL-FOOTER': () => [this.elementRef.nativeElement],
+    'KIRBY-PAGE-TITLE': () =>
+      [this.ionTitleElement.nativeElement, this.contentTitle?.nativeElement].filter(
+        (e) => e !== undefined
```

Comment:
could we enhance the readability a bit by writing out `e` as `element`? At the very first glance this read as some kind of error handling to me.

Reply:
Good point - i've changed it to element!